controller/src/test/java/io/pravega/controller/store/index/ZkHostIndexTest.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
```

Comment:
This is the wrong license header, please use the one specified in the project. Also, please remove the copyright year.

Reply:
oops. looks like some classes still have the old license header. I copy pasted from some class. opened an issue (#3989) to clean up old license headers .